components/eamxx/cmake/ScreamUtils.cmake

```diff
@@ -45,40 +45,27 @@ set(SCREAM_CUT_TEST_MV_ARGS ${CUT_TEST_MV_ARGS})
 # Scream always excludes the ekat test session since it has its own
 list(REMOVE_ITEM SCREAM_CUT_EXEC_OPTIONS EXCLUDE_TEST_SESSION)
 
-# Libs are a position arg for SCREAM, not an optional arg like in EKAT
-list(REMOVE_ITEM SCREAM_CUT_EXEC_MV_ARGS LIBS)
-
 ###############################################################################
-function(CreateUnitTestExec exec_name test_srcs scream_libs)
+function(CreateUnitTestExec exec_name test_srcs)
 ###############################################################################
-  cmake_parse_arguments(cute "${SCREAM_CUT_EXEC_OPTIONS}" "${SCREAM_CUT_EXEC_1V_ARGS}" "${SCREAM_CUT_EXEC_MV_ARGS}" ${ARGN})
-  CheckMacroArgs(CreateUnitTestExec cute "${SCREAM_CUT_EXEC_OPTIONS}" "${SCREAM_CUT_EXEC_1V_ARGS}" "${SCREAM_CUT_EXEC_MV_ARGS}")
-
-  separate_cut_arguments(cute "${SCREAM_CUT_EXEC_OPTIONS}" "${SCREAM_CUT_EXEC_1V_ARGS}" "${SCREAM_CUT_EXEC_MV_ARGS}" options)
-
-  set(TEST_INCLUDE_DIRS
```

Comment:
`SCREAM_INCLUDE_DIRS` is never used. And I thought that current source/binary dirs can be added when needed, via the `INCLUDE_DIRS` optional arg.